lib/user_agent/browsers/pocket_casts.rb

```diff
@@ -0,0 +1,91 @@
+class UserAgent
+  module Browsers
+    # Pocket Casts BMID/E678F58F21
+    # PocketCasts/1.0 (Pocket Casts Feed Parser; +http://pocketcasts.com/)
+    # Shifty Jelly Pocket Casts, Android v4.5.3
+    # Shifty Jelly Pocket Casts, iOS v4.3
+    # Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Pocket Casts/1.1 Pocket Casts/1.1
+    class PocketCasts < Base
+      def self.extend?(agent)
+        agent.detect { |useragent| useragent.product =~ /PocketCasts/ } || agent.to_s =~ /Pocket Casts/
```

Comment:
create/use constants for the regular expressions. I guess we cannot rely on having `Regex#match?` (ruby 2.4) available as that would be faster. Otoh, this is our fork.